cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -78,10 +78,10 @@
              %}
             {% set value = global_dict.expandable %}
             {% include 'molecules/expandable.html' with context %}
+            {% set value = None if prototype %}
```

Comment:
I'm actually unclear when the prototype pages would be viewed without the `prototype` value being present. The purpose of this is when setting the mock data in the `value` for the prototype pages, if you don't null it out the next atomic element gets the date from the previous atomic element. So like the text intro gets expandable data, etc.